examples/basic/main.tf

```diff
@@ -0,0 +1,35 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+locals {
+  env       = "sandbox"
+  name      = "example"
+  namespace = "aws-rhythmic-sandbox"
+  owner     = "Rhythmictech Engineering"
+
+  extra_tags = {
+    delete_me = "please"
+    purpose   = "testing"
+  }
+}
+
+module "tags" {
+  source  = "rhythmictech/tags/terraform"
+  version = "1.0.0"
+
+  names = [local.name, local.env, local.namespace]
+
+  tags = merge({
+    "Env"       = local.env,
+    "Namespace" = local.namespace,
+    "Owner"     = local.owner
+  }, local.extra_tags)
+}
```

Comment:
Don't think we need the tags module in the example

Reply:
Yeah we do - we're demonstrating the Rhythmic way of doing things and getting a few downloads on our registry page

Reply:
fair enough